susemanager-utils/testing/automation/push-to-obs.sh

```diff
@@ -22,10 +22,11 @@ help() {
   echo "  -n  If used, update PROJECT instead of the projects specified with -d,"
   echo "      for example, if you want to package only the changes from a PR on"
   echo "      a separate project"
-  echo "  -e  If used, when checking out projects from obs, links will be expanded. Useful for comparing packages that are links" 
+  echo "  -e  If used, when checking out projects from obs, links will be expanded. Useful for comparing packages that are links"
```

Comment:
I am now completely confused. Where did the option for parallel is?

I don't see `-d` at the docker calls either, what happened :-?

Reply:
Hi,

the parallelism is accomplished with using amperstand and wait. Using the -d was redundant, was not printing the logs in stderr nor stdout, and, overall, wait was not waiting for them to finish.

I want to use wait because I want to ensure the process does not end until all packages have been pushed to obs.

I updated the title to reflect this is about parallelism.

Sorry I removed the option "x" and made parallelism mandatory cause the "x" option was not working well for testing this PR with the current jenkins job implementation.

My question remains, should we make it default, or should the parallelism be optional?

Reply:
I'd certainly make it optional with `-x`. What's the problem with the Jenkins job implementation?

Reply:
Ok, I'll do that. The problem is that the current job can't have the "-x" without having this code merged first, it is a "fish eating its tail" problem ;) . But I can certainly do this once we are ready to merge it.

Reply:
I don't understand, what's wrong with the jobs? If we make parallelism optional with `-x`, the jobs will work before the PR is merged and after the PR is merged, as parallelism won't be used because `-x` is not used.

When the PR is merged, then you can configure them to use `-x`.

Where's the "fish eating its tail" problem :-?